src/content/posts/FSCS-2021-Intro-Bofbof.md

```diff
@@ -0,0 +1,282 @@
+---
+title: "write-up : bofbof (FCSC intro 2021)"
+date: 2024-02-14
+#draft: true
+draft: false
+tags: ["iscsc","blog","posts"]
+---
+
+## Présentation du challenge
+
+Le challenge contient :  
+`docker-compose.yml` : un fichier de configuration permettant d'accéder à un conteneur docker en local  
+`bofbof` : un executable ELF x64, dynamically linked & not stripped 

```

Comment:
could you add how to start the docker compose ?

Reply:
[d85247f](https://github.com/iScsc/blog.iscsc.fr/pull/14/commits/d85247f685357ead81645c6dc23c32b038750d1a)